### ISSUE TITLE
[Element] Add part to pagination bullets and active pagination bullet

### ISSUE THREAD

**Issue by strarsis** [nolimits4web/swiper#6717]
### Clear and concise description of the problem

For specific special styles to the swiper pagination bullets, each bullet needs to be an additional part.
Additionally, the active bullet needs to have a separate part name.

### Suggested solution

Each bullet gets the part name `pagination-bullet`.
The active bullet gets the part name `pagination-bullet--active`.

### Alternative

There are no real alternatives to adding those parts in core, except changing the swiper elements using JavaScript, which is neither robust nor having good performance.

### Additional context

E.g. `aspect-ratio`; `mask`; `transition` and `background`.

Component parts allow adding those styles cleanly and efficiently.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [X] I'm willing to open a PR

**Comment by timeisgolden** [comment]
Hello, Team, I am trying to change the pagination bullet background color, 
my code is the following.
`
swiper-container::part(pagination) {
  .swiper-pagination {
    background-color: red !important;
    .swiper-pagination-bullet {
      background-color: white !important;
    }
    .swiper-pagination-bullet-active {
      background: white !important;
    }
  }
}
`

**Comment by strarsis** [comment]
@timeisgolden: Does that work for you? I tried this without explicit `part` on the bullet (and bullet state) and it does not appear to work from a parent `parent`.

**Comment by strarsis** [comment]
Correct, the bullet – and ideally the state (active or not) – should be added as `part`s by Swiper Element itself.

**Comment by timeisgolden** [comment]
@strarsis but I can't change the style of the button(active bullet) with my above code.
Can you share a sample code to change the style of active bullet? especially, I want to change the background of active bullet.
